server/middleware/validationMiddleware.ts

```diff
@@ -14,6 +14,11 @@ function validationMiddleware(type?: new () => object): RequestHandler {
 
     const classType = (getAdditionalConditionByCode(req.body.code)?.type as ClassConstructor<object>) || type
 
+    // Must add the uploaded filename to the req.body for file upload types to validate that a file has been selected
+    if (req.file && classType?.name === 'OutOfBoundsRegion') {
+      req.body = { ...req.body, outOfBoundFilename: req.file.originalname }
+    }
+
     const validationScope = plainToClass(classType, { ...req.body, licence }, { excludeExtraneousValues: false })
```

Comment:
Instead of explicitly harcoding `OutOfBoundsRegion`, you could add `req.file` to the validation scope here? and add a validator annotation to `outOfBoundsFilename`?